src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
```

Comment:
```suggestion
  const inversedRobot = {};
```
Let's specify the name